src/Tasks/Common/Resources/xlf/Strings.ja.xlf~RF1ea49cc.TMP

```diff
@@ -0,0 +1,914 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
src/Tasks/Common/Resources/xlf/Strings.ja.xlf~RF1ea49cc.TMP looks like a name for a temporary file; did you add this by accident?

There is the older Strings.es.xlf~RF13d378ed.TMP as well.

Reply:
Yeah this looks like a mistake, filed https://github.com/dotnet/sdk/pull/19688 to delete them.

Reply:
Thanks for catching that.  I've submitted #19694 to fix.